server/lib/user-update.js

```diff
@@ -7,7 +7,7 @@ const isGroup = require("./utils/is-group-trait");
     group,
     (m, v, k) => {
       const n = key ? `${key}/${k}` : k;
-      if (isGroup(v)) {
+      if (isGroup(v) && !n.includes("/")) {
```

Comment:
I don't remember what was the issue with this exactly... but I remember that it was potentially a breaking change, right?  Is it ok to push it to master now ?

Reply:
it's in a commented out area...